admin_manual/configuration_server/occ_command.rst

```diff
@@ -672,20 +672,31 @@ search path. If not using ``--quiet``, statistics will be shown at the end of
 the scan::
 
  sudo -u www-data php occ files:scan --help
-   Usage:
-   files:scan [-p|--path="..."] [-q|--quiet] [-v|vv|vvv --verbose] [--all]
-   [user_id1] ... [user_idN]
+ Description:
+   rescan filesystem
+
+ Usage:
+   files:scan [options] [--] [<user_id>...]
 
  Arguments:
-   user_id               will rescan all files of the given user(s)
+   user_id                  will rescan all files of the given user(s)
 
  Options:
-   --path                limit rescan to the user/path given
-   --all                 will rescan all files of all known users
-   --quiet               suppress any output
-   --verbose             files and directories being processed are shown
-                         additionally during scanning
-   --unscanned           scan only previously unscanned files
+       --output[=OUTPUT]    Output format (plain, json or json_pretty, default is plain) [default: "plain"]
+   -p, --path=PATH          limit rescan to this path, eg. --path="/alice/files/Music", the user_id is determined by the path and the user_id parameter and --all are ignored
+       --generate-metadata  Generate metadata for all scanned files
```

Comment:
should we add a short note about EXIF extraction to make it less abstract ? or would this belong in a different section ?